src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -302,10 +304,29 @@ public Pod build() {
         }
 
         // default jnlp container
-        Optional<Container> jnlpOpt = pod.getSpec().getContainers().stream().filter(c -> JNLP_NAME.equals(c.getName()))
-                .findFirst();
-        Container jnlp = jnlpOpt.orElse(new ContainerBuilder().withName(JNLP_NAME)
-                .withVolumeMounts(volumeMounts.values().toArray(new VolumeMount[volumeMounts.values().size()])).build());
+        Optional<Container> jnlpOpt = pod.getSpec().getContainers().stream().filter(c -> JNLP_NAME.equals(c.getName())).findFirst();
+        Container jnlp = jnlpOpt.orElse(new ContainerBuilder().withName(JNLP_NAME).withVolumeMounts(
+                volumeMounts.values().toArray(new VolumeMount[volumeMounts.values().size()])).build());
+
+        // JENKINS-71639 - Inject `securityContext` in `jnlp` container definition when option activated
```

Comment:
I think you want to use https://github.com/jenkinsci/kubernetes-plugin/blob/746e28c7546b7156692a162927cd52129762d8fb/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pod/decorator/PodDecorator.java#L10-L12 rather than hard-coding this here. In fact this sort of thing could be done by a separate plugin.

Reply:
Oh that's indeed what I wanted to use! Thanks a lot for the pointer :heart: 

I refactored the code to use it.